### PR TITLE
Add scancode

### PR DESCRIPTION
**Pull request by style95**

### CODE REVIEW

File: resource/template/actioncode/sample.js [hunk 1]
```diff
@@ -1,3 +1,20 @@
+/*
```
Comment (upgle) [inline resource/template/actioncode/sample.js:1]: I'd liked to know the license is required for a sample code.
I am not sure about the licensing policy.

Comment (style95) [reply resource/template/actioncode/sample.js:1]: This is according to the `scancode.sh`.
Even test codes in the openwhisk repo have a license header.
https://github.com/apache/openwhisk/blob/master/tests/dat/actions/echo.js

Comment (upgle) [reply resource/template/actioncode/sample.js:1]: This is not a test code. When a user creates a new action with the plugin, this sample code is used.

Comment (style95) [reply resource/template/actioncode/sample.js:1]: I feel it's reasonable to add a license header as this code is also provided by asf?

**Review by upgle — APPROVED** [review]
LGTM

**Pull request merged**